### DISCUSSION TITLE
[Question] Are we going to support asynchronous API calls with this new client

### DISCUSSION THREAD

**Discussion by souvik-databricks** [databricks/databricks-sdk-py#75] [Ideas]
This is an ask from one of my customers if we are planning to support a asynchronous client for our SDK?

As the current client
```python
from databricks_cli.sdk.api_client import ApiClient
``` 
doesn't supports asynchronous calls

**Comment by nfx** [comment]
@souvik-databricks, perhaps, eventually. It's definitely not the initial focus, because auth layer also has to be async.
